xarray/tests/test_accessor_str.py

```diff
@@ -1,3 +1,5 @@
+# -*- coding: utf-8 -*-
```

Comment:
It is used by a variety of tools and text editors besides Python, so I would prefer to keep it in.

Reply:
the magic comment does not really hurt since `utf-8` is the default source code encoding for python 3. However, we don't have editor specific comments ([PEP 263](https://www.python.org/dev/peps/pep-0263/) calls the format you used the "emacs style") anywhere else so for consistency it might be better to remove it.

Reply:
Do we still want this to be addressed before merging?

Reply:
Fixed.